Snabble/Core/QRCode/QRCodeGenerator.swift

```diff
@@ -8,15 +8,18 @@ import Foundation
 
 /// generate QR code(s) for offline payment
 public struct QRCodeGenerator {
-    private var cart: ShoppingCart
-    private var config: QRCodeConfig
+    private let cart: ShoppingCart
+    private let config: QRCodeConfig
+    private let processId: String?
```

Comment:
Reines Interesse: Warum sind die `private` obwohl der Entwickler sie in der `init` mitgibt?

Reply:
Kein besonderer Grund, aber was von aussen nicht benötigt/referenziert wird sollte IMO auch nicht sichtbar sein